### ISSUE TITLE
Different lambda values

### ISSUE THREAD

**Issue by heydarshahi** [RenYang-home/OpenDVC#20]
Hi! 

I have a question regarding the pre-trained models: did you train a model from scratch for each lambda value?  

Many thanks,

Amin

**Comment by RenYang-home** [comment]
hi, what do you mean "from scratch"?

--
Ren Yang
https://renyang-home.github.io/

Doctoral student
Computer Vision Laboratory | ETH Zürich
Sternwartstrasse 7 | ETF D 114.2 | 8092 Zürich, Switzerland

**Comment by heydarshahi** [comment]
I mean did you train a completely separate model for each lambda value? 

I was thinking maybe if I load a pretrained model but change the lambda value in the loss function, and continue the training, I might come up with a new model sooner than training the model from scratch with random weights using the new lambda value...

**Comment by RenYang-home** [comment]
I train the models of different lambda seperately, but the four MS-SSIM models are finetuned from the four PSNR models, respectively

--
Ren Yang
https://renyang-home.github.io/

Doctoral student
Computer Vision Laboratory | ETH Zürich
Sternwartstrasse 7 | ETF D 114.2 | 8092 Zürich, Switzerland

**Comment by heydarshahi** [comment]
Thanks for the response 😊
